MensaHub-DataDispatcher/src/main/java/de/olech2412/mensahub/datadispatcher/data/leipzig/leipzigDispatcher/CollaborativeFilteringLeipzigDispatcher.java

```diff
@@ -151,11 +151,10 @@ private void sendCollabPushNotification(LocalDate tomorrow, MailUser mailUser, M
      * @return The message
      */
     private String buildMealMessage(MailUser mailUser, Mensa mensa, LocalDate tomorrow) {
-        StringBuilder message = new StringBuilder();
-        message.append("Hallo ").append(mailUser.getFirstname()).append(",\n");
-        message.append("klicke hier für deine Empfehlungen für morgen, den ").append(tomorrow).append(" in der ").append(mensa.getName()).append("\n");
-        message.append("Guten Appetit!");
-        return message.toString();
+        String message = "Hallo " + mailUser.getFirstname() + ",\n" +
```

Comment:
das ist für mich weniger readable

Reply:
append für jede Zeile der nachricht

Reply:
> append für jede Zeile der nachricht

StringBuilder message = new StringBuilder();
        message.append("Hallo ").append(mailUser.getFirstname()).append(",\n");
        message.append("klicke hier für deine Empfehlungen für morgen, den ").append(tomorrow).append(" in der ").append(mensa.getName()).append("\n");
        message.append("Guten Appetit!");
        return message.toString();

Reply:
das hat intellij beim code refactoring selbst gemacht, habe an deinem code per hand nichts geändert

Reply:
außer merge konflikte